release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1263,6 +1291,16 @@ Once the firewall rule of the machine with the missing infrastructure ID is remo
 
 * The `opm alpha bundle build` command fails on Windows 10. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883773[*BZ#1883773*])
 
+* When you install Agones using Helm package manager and try to examine the chart resources in your namespace using the *Developer* perspective, you see an error message instead of the resource details.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1866087[BZ#1866087])
+
+* When you select a deployment in the *Topology* view, click *Actions* -> *Edit <deployment_name>*, and then modify it; the modified `Deployment` YAML file overwrites or removes the volume mounts in the `Pod Template spec`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1867965[BZ#1867965])
+
+* No template instantiation success or failure message is displayed in the *Developer* perspective when you use the *Add* -> *From Catalog* option, filter by *Template*, select a template, and then instantiate it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1876535[1876535])
+
+* Pipeline runs with skipped tasks incorrectly show the tasks as *Failed*. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880389[1880389])
+
+// confirmation on this one needed* The Kiali link can return false positive responses as the Service Mesh operator <replace with Elastic Search operator?> is unavailable in the new bundle format yet. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1866035[1866035])
+
```

Comment:
this would be closed IMO cc @gajanan-more

Reply:
Ack, removing this.